queries/unison/highlights.scm

```diff
@@ -0,0 +1,75 @@
+;; Primitives
+(comment) @comment
+(nat) @number
+(unit) @constant.builtin
+(literal_char) @character
+(literal_text) @string
+(literal_boolean) @boolean
+
+;; Keywords
+[
+  (kw_forall)
+  (unique_kw)
+  (type_kw)
+  (kw_equals)
+  (do)
+] @keyword
```

Comment:
forall and do sound like `@repeat`

equals sounds like `@keyword.operator`

Reply:
`forall` - is a type level thing in unison (https://www.unison-lang.org/learn/language-reference/types/, https://www.unison-lang.org/learn/language-reference/scoped-type-variables/)
`do` - introduces a delayed computation block https://www.unison-lang.org/learn/language-reference/delayed-computations

I think you are right about equals, will update.

Reply:
Thanks, not familiar w/ the language so I just go off of preconceived notions about certain keywords I see often :)